.gitignore

```diff
@@ -20,6 +20,8 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
+*.xcodeproj/
+DerivedData/
```

Comment:
1. Exclude the Xcode project directory. 
2. Exclude the default directory created for object files and other build data.

Reply:
We have `ios/covidsafe.xcodeproj`  checked in because we have custom stuff in it.